packages/ui/src/lib/layouts/FormPage.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+import { CloseButton, LinearProgress, Link } from '@podman-desktop/ui-svelte';
```

Comment:
Maybe we should not import itself as a package ?

Reply:
what is strange is why it's only reproducing on the CI, I would hope that developers have the same failure

Reply:
maybe try to delete the export/dist folder, and you _should_ be able to reproduce ?

Reply:
Or delete node_modules? i.e. you've already done a yarn install and have a local cache of what it will build here.